src/app/create-template/nhs-app/page.tsx

```diff
@@ -0,0 +1,9 @@
+export default function CreateNhsAppTemplate() {
+  return (
+    <>
+        <h2 className='nhsuk-heading-l' data-testid='page-sub-heading'>
+            Placeholder NHS App page
```

Comment:
Can we move the text to the content file

Reply:
these are just placeholder pages to prove the server action redirect works, when we pick up the next tickets to actually develop these pages we'll set up the text in the content file

Reply:
Lets leave the placeholders where they are for now as they will soon be deleted.